lib/fluent/plugin/in_prometheus_monitor.rb

```diff
@@ -65,11 +71,18 @@ def start
 
     def update_monitor_info
       @monitor_agent.plugins_info_all.each do |info|
+        label = labels(info)
+
         @monitor_info.each do |name, metric|
           if info[name]
-            metric.set(labels(info), info[name])
+            metric.set(label, info[name])
           end
         end
+
+        if timekeys = info["buffer_timekeys"]
```

Comment:
When the value is empty array, an exception occurs in set.

Reply:
I've pushed a new version that fixes this, but this also means that when the buffer is empty, these metrics keep the values they last had when the buffer had data in it. There doesn't seem to be a way to unset these metrics once they've been set. Can you think of a better way to handle this case?

Reply:
probably it's empty until the first flush so I always experienced an exception here like this
```Unexpected error raised. Stopping the timer. title=:in_prometheus_monitor error_class=ArgumentError error="value must be a number"```